### PR TITLE
LPS-179290 Stop propagation so the card does not activate when pressing the dropdown buttons

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js [hunk 1]
```diff
@@ -365,6 +366,72 @@ export function MenuItem({
 									</div>
 								</ClayLayout.ContentCol>
 
+								<AddItemDropDown
+									className="position-absolute site_navigation_menu_editor_MenuItem-add-button-dropdown top-button"
```
Comment (sandrodw3) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js]: Maybe we could add the common classes inside the component 😄

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js [hunk 2]
```diff
@@ -331,85 +331,92 @@ export function MenuItem({
...
-												title={Liferay.Language.get(
-													'add-item'
-												)}
-											/>
-										}
-									/>
-								)}
-
-								<ClayLayout.ContentCol gutters>
+								<div
```
Comment (sandrodw3) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js:334]: We could move the FF condition outside the div, so we only have one 😛

**Review by sandrodw3 — APPROVED** [review]
Weee LGTM 😄 works super fine!

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js]: Those ones only apply here, there are other usages of the dropdown where it doesn't apply :peeposhy:

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js (hunk 2 above)
Comment (victorg1991) [reply modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js:334]: Fair enough :p

**Comment by victorg1991** [comment]
ci:reevaluate:1327401_8856

**Comment by victorg1991** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
failures seem to be related to the changes

**Comment by victorg1991** [comment]
@Tim-Cao could you help me identifying what was broken, please? 😇

**Comment by Tim-Cao** [comment]
Hi @victorg1991, the remove button is invisible when hover on it.
image

**Comment by p2kmgcl** [comment]
https://github.com/liferay-echo/liferay-portal/pull/11659

**Pull request closed without merging**